src/parser/smt2/smt2.cpp

```diff
@@ -1325,6 +1325,12 @@ Term Smt2State::applyParseOp(const ParseOp& p, std::vector<Term>& args)
     Trace("parser") << "applyParseOp: return store all " << ret << std::endl;
     return ret;
   }
+  else if (p.d_kind == APPLY_TESTER || p.d_kind == APPLY_UPDATER)
+  {
+    Term iop = mkIndexedOp(p.d_kind, {p.d_name}, args);
+    kind = p.d_kind;
+    args.insert(args.begin(), iop);
```

Comment:
Why is the end here is a term iop? Did you mean args.end()?

Reply:
This code is determining how to construct the operator, which should be passed as the first child to `mkTerm`. It is prepended to args.

Reply:
Sorry, my bad. You are just inserting the term at the beginning. 
I thought this is the overloaded version of insert that copies multiple elements.